api/jreleaser-model-api/src/main/java/org/jreleaser/model/api/announce/MastodonAnnouncer.java

```diff
@@ -29,5 +31,5 @@ public interface MastodonAnnouncer extends Announcer {
 
     String getAccessToken();
 
-    String getStatus();
```

Comment:
Please keep the single `status` property otherwise this becomes a break in binary compatibility. The `org.jreleaser.model.api.announce.TwitterAnnouncer` kept its `status` property as well.